content/academy/apify_platform/deploying/deploying.md

```diff
@@ -46,46 +46,12 @@ That's it! the actor should now pull its source code from the repo and automatic
 
 ## [](#with-apify-cli) Without a Github repository (using the Apify CLI)
 
-The [Apify CLI](https://docs.apify.com/cli) helps you create, develop, build and run Apify actors, and manage the Apify cloud platform from any computer. It can be used to automatically generate the boilerplate for different types of projects, initialize projects, remotely call actors on the platform, and run your own projects.
+> If you don't yet have the Apify CLI, learn how to install it and log in by following along with [this brief lesson]([Apify CLI]({{@link tools/apify_cli.md}})) about it.
 
-### [](#installing-cli) Installing
+If you're logged in to the Apify CLI, the `apify push` command can be used to push the code straight onto the Apify platform from your local machine (no Github repository required), where it will automatically be built for you. Prior to running this command, make sure that you have an **apify.json** file at the root of the project. If you don't already have one, you can use `apify init PROJECT_NAME` to automatically generate one for you.
```

Comment:
I think you can just do `apify init` without the name. It will take the name from the folder.

Also, might be worth mentioning how you can use `.gitignore` to exclude files from being pushed. When you `apify push` without `.gitignore`, the full folder contents will be pushed. This means that most likely even apify_storage and node_modules. We don't want that.

Next, I would add some screenshot of the pushed actor on the platform with multifile editor, so that the users connect the push with the UI.

Reply:
You can, true. You can do `apify init .`